metadata-ingestion/src/datahub/specific/datajob.py

```diff
@@ -504,7 +504,7 @@ def add_output_dataset_field(
             DataJobInputOutput.ASPECT_NAME,
             "add",
             path=f"/outputDatasetFields/{self.quote(output_urn)}",
-            value=output_edge,
```

Comment:
Does the interface still make sense? Like should we change this to only take an urn instead of an edge?

Reply:
Yeah this method doesn't need to take in an edge when it's not using it.